app/templates/@layout.latte

```diff
@@ -63,6 +63,7 @@
 		{block main}
 		<div class="main" lang="en">
 			<div class="content has-sidebar">
+				<div class="flash error">See our new modern addons portal <a href="http://bit.ly/cmtte">componette.com</a>. Completely automatic.</div>
```

Comment:
I think that something like `https://componette.com/?utm_source=referrer&utm_medium=link&utm_campaign=addons.nette.org` is perfectly sufficient :) But either way is better.